### ISSUE TITLE
SqlClient 4.4.1 not published to NuGet

### ISSUE THREAD

**Issue by kenleese** [dotnet/corefx#25834]
SqlClient 4.4.1 resolves https://github.com/dotnet/corefx/issues/13422.
The November Update release notes indicate SqlClient 4.4.1 is released https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0.3.md
Howvere the latest version on NuGet appears to be 4.4.0 https://www.nuget.org/packages/System.Data.SqlClient/
Am I misunderstanding?
cc @saurabh500

**Comment by saurabh500** [comment]
@corivera what's going on in this case?

**Comment by corivera** [comment]
@weshaggard There's a SqlClient 4.4.1 on myget (https://dotnet.myget.org/feed/dotnet-core/package/nuget/System.Data.SqlClient/4.4.1), but not on nuget. Is this expected?

**Comment by weshaggard** [comment]
That one does look like it was missed to go on nuget.org. @dagood could you please take a look at get this package published to nuget.org?

**Comment by dagood** [comment]
That SqlClient 4.4.1 on MyGet has a different commit hash in its version.txt than the Microsoft.NETCore.Platforms 2.0.1 package that we shipped for release 2.0.3. The drop where I pulled Platforms 2.0.1 from didn't have a stable SqlClient package.

That doesn't prevent us from pushing from MyGet to the NuGet Gallery, but I'd like to understand the situation better first such as tracking down which build that 4.4.1 came from. I'm looking into it.
